d3a_api_client/redis_aggregator.py

```diff
@@ -0,0 +1,207 @@
+import logging
+import uuid
+import json
+from redis import StrictRedis
+from d3a_interface.utils import wait_until_timeout_blocking
+from concurrent.futures.thread import ThreadPoolExecutor
+from d3a_api_client.constants import MAX_WORKER_THREADS
+from threading import Lock
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
+
+
+class RedisAPIException(Exception):
+    pass
+
+
+class RedisAggregator:
+
+    def __init__(self, aggregator_name, autoregister=False,
```

Comment:
Pleas remove the `autoregister` parameter totally as it gives a false impression. The aggregator does not register. Thanks!